parsers/browscap-js-1/package.json

```diff
@@ -10,6 +10,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "browscap-js": "^1.0"
+    "browscap-js": "^1.11.6027"
```

Comment:
This change shouldn't be necessary no?  `^1.0` should pull down anything in the 1.x version range right? Were you having issues with it pulling down the latest version?

Reply:
fixed